newIDE/app/src/Utils/ZoomUtils.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+const stepZoomFactor = Math.pow(2, 1 / 8);
+export const zoomInFactor = stepZoomFactor;
+export const zoomOutFactor = Math.pow(stepZoomFactor, -1);
+const continuousZoomFactor = Math.sqrt(stepZoomFactor);
```

Comment:
As it's a constant that will only be calculated once `2 ** (1/16)` will be easier to understand as an half step.